testing/python_test

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -e
+
+ROOT=$(realpath $(dirname $0)/..)
+PREFIX=
+PYTHON=python3
+EXTARGS=
+coverage=
+
+if [ "$1" == coverage ]; then
+    coverage=y
+    PYTHON=coverage3
+    EXTARGS="run --source $ROOT/forch"
+    shift
+fi
+
+if [ $# != 1 ]; then
+    echo $0 [test_name]
+    false
+fi
+
+TEST_NAME=$1
+shift
+
+TEST_DIR=testing/python_lib
```

Comment:
Should we just rename it to python_test since we have the test suite files themselves here?

Reply:
There's already a python_test in that directory (this file itself), so can't use that name...